tutorials/tutorial04_visualize.ipynb

```diff
@@ -111,7 +111,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "! tensorboard /ray_results/experiment_dir/result/directory"
```

Comment:
BTW, the '!' shoube be '$'

Reply:
Hey @CYBruce , thanks for doing this! What do you mean by the '$' though?

Reply:
Sorry, I think I made a wrong suggestion. I just don't understand what does '!' mean.

Reply:
Ah, no worries. Looks good though! Will accept and merge now